src/tests/accomodation/accommodation.test.js

```diff
@@ -0,0 +1,69 @@
+import chai, { expect } from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../../app';
+import accommodation from '../fixtures/accommodation.fixture';
+import { loggedInToken, createUsers } from '../fixtures/users.fixture';
+
+chai.should();
+chai.use(chaiHttp);
+
+describe('Test booking accommodation:', () => {
+  before(async () => {
+    await createUsers();
+  });
+  it('Should return status code of 201 on successful accommodation creation', (done) => {
+    chai.request(app)
+      .post('/api/book-accommodation')
+      .set('Authorization', loggedInToken)
+      .send(accommodation)
+      .end((err, res) => {
+        expect(res).to.have.status(201);
```

Comment:
Do not only check for status code. Check that the right response structure is being returned.